deployment/setup_20.04.sh

```diff
@@ -17,21 +18,27 @@ apt install -y \
     libssl-dev libyaml-dev libreadline-dev gnupg2 nginx redis-server \
     redis-tools postgresql postgresql-contrib certbot \
     python3-certbot-nginx nodejs yarn patch ruby-dev zlib1g-dev liblzma-dev \
-    libgmp-dev libncurses5-dev libffi-dev libgdbm6 libgdbm-dev
+    libgmp-dev libncurses5-dev libffi-dev libgdbm6 libgdbm-dev nginx-full
 
 adduser --disabled-login --gecos "" chatwoot
 
-sudo -i -u chatwoot bash << EOF
 gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+gpg2 --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
 curl -sSL https://get.rvm.io | bash -s stable
-EOF
+addgroup chatwoot rvm
 
 pg_pass=$(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 15 ; echo '')
 sudo -i -u postgres psql << EOF
 \set pass `echo $pg_pass`
 CREATE USER chatwoot CREATEDB;
 ALTER USER chatwoot PASSWORD :'pass';
 ALTER ROLE chatwoot SUPERUSER;
+UPDATE pg_database SET datistemplate = FALSE WHERE datname = 'template1';
```

Comment:
Would it be best to have this change in another PR ?

Reply:
Why ?

Reply:
Unrelated change ?

Reply:
No, otherwise there is an error on the type of encoding used